config/routes.rb

```diff
@@ -30,6 +30,8 @@
 
   get '/search', to: "search#search"
 
+  get '/about', to: "about#about"
```

Comment:
Could we change this method name to `index` to make it more resource like? 
Also I think you'll have to move `apps/views/logix/about.html.erb` to its appropriate location.

Reply:
Actually `app/views/logix/about.html.erb` has been already moved to `app/views/about/about.html.erb` (now changed to `app/views/about/index.html.erb`)
I just changed method name to `index`  in https://github.com/CircuitVerse/CircuitVerse/pull/728/commits/9907b986a3e8c5a875ca490f1841a96081ddb6f9